aea/helpers/io.py

```diff
@@ -87,3 +89,28 @@ def open_file(
     return actual_wrapped_function(
         file, mode=mode, buffering=buffering, encoding=encoding, errors=errors
     )
+
+
+def to_csv(package_hashes: Dict[str, str], path: Path) -> None:
```

Comment:
Since this is a generic helper method the arg name should be more generic: `package_hashes` -> something more generic